CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to Zephyr
+
+### Developer Certification of Origin (DCO)
+
+Licensing is very important to open source projects. It helps ensure the
+software continues to be available under the terms that the author desired.
+
+Zephyr uses [the Apache 2.0 license]
```

Comment:
to get this link to display correctly (in markdown) the close ``]`` and opening ``(`` need to be adjacent:

     Zephyr uses [the Apache 2.0 license](
     https://github.com/zephyrproject-rtos/zephyr/blob/master/LICENSE) to strike a